core/src/main/java/io/spine/core/Events.java

```diff
@@ -162,6 +162,20 @@ public static UserId getActor(EventContext context) {
         return id;
     }
 
+    /**
+     * Obtains origin ID of the Event — the id of the original message that spawned this event.
```

Comment:
This is both wrong (as you used "origin", meaning something else in this context) and wrong (it should be `ID`, not `id`; we also don't use `—` symbols as much).

I have rewritten it like this. 

```
Obtains the ID of the root command, which lead to this event.

<p> In case the passed {@code Event} instance is a reaction to another {@code Event}, the identifier of the very first command in this chain is returned.
```